cdc/pkg/version/check.go

```diff
@@ -65,6 +65,7 @@ func removeVAndHash(v string) string {
 	}
 	v = versionHash.ReplaceAllLiteralString(v, "")
 	v = strings.TrimSuffix(v, "-dirty")
+	v = strings.TrimPrefix(v, "cdc-") // tag is named with cdc-vx.x.x
```

Comment:
Is tikv-cdc better ?

Reply:
It depends on the release tag naming style. Now we use `cdc-vx.x.x` and `br-vx.x.x`, which omit the `tikv` prefix.

Reply:
Get it.